internal/controller/rbac/namespace/reconciler.go

```diff
@@ -210,9 +213,15 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 		}
 
 		log.Debug("Applied RBAC Role")
```

Comment:
would it be useful in this debug log statement to include the name of the RBAC role that was just applied?